### PR TITLE
develop merge

### PR DESCRIPTION
**Pull request by ROSHiix**

### CODE REVIEW

File: app/src/main/java/com/ravenwallet/tools/sqlite/BRSQLiteHelper.java [hunk 1]
```diff
@@ -44,8 +44,8 @@ public static BRSQLiteHelper getInstance(Context context) {
         return instance;
     }
 
-    public static final String DATABASE_NAME = "ravenwallet.db";
-    private static final int DATABASE_VERSION = 15;
```
Comment (dustinlewis) [inline app/src/main/java/com/ravenwallet/tools/sqlite/BRSQLiteHelper.java]: @ROSHiix Why the database version change (downgrade) and the database name change? This seems problematic. Have you tried an upgrade scenario with these changes?

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -558,9 +557,7 @@
     <!-- Updating webview message -->
     <string name="Webview.updating">Updating...</string>
     <!-- Welcome screen text. (?) will be replaced with the help icon users should look for. -->
-    <string name="Welcome.body">Ravencoin Wallet has changed its name to RVN, with a brand new look and some new features.\n\nIf you need help, look for the (?) in the top right of most screens.</string>
```
Comment (dustinlewis) [inline app/src/main/res/values/strings.xml]: @ROSHiix Are these strings just not used anymore? Why leave the comment text about the strings that are no longer there?

File: app/src/main/java/com/ravenwallet/tools/sqlite/BRSQLiteHelper.java (hunk 1 above)
Comment (ROSHiix) [reply app/src/main/java/com/ravenwallet/tools/sqlite/BRSQLiteHelper.java]: The database downgrade was done when we moved from ravencoin wallet classic to the one with assets.

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (ROSHiix) [reply app/src/main/res/values/strings.xml]: This is not used anymore, I must remove it from code first.

**Pull request merged**